config/.prettierrc

```diff
@@ -1,5 +1,6 @@
 {
 	"singleQuote": true,
 	"trailingComma": "all",
-	"useTabs": true
+	"useTabs": true,
+	"arrowParens": "avoid"
```

Comment:
I actually think it makes sense to always include these, for the very reasons that prettier changed the default

Reply:
Works for me, will update

Reply:
Agreed, I find it slightly better to always have parens, so adding additional parameters to the arrow function after it was defined, or specifying typings doesn't require manually adding parens.